dagmc_stats/DagmcQuery.py

```diff
@@ -1,11 +1,18 @@
 from pymoab.rng import Range
 from pymoab import core, types
+import pandas as pd
+import numpy as np
 import warnings
 
 class DagmcQuery:
     def __init__(self, dagmc_file, meshset=None):
         self.dagmc_file = dagmc_file
         self.meshset = meshset
```

Comment:
We should we probably check here `if meshset is None` and set `self.meshset` to the rootset if it is. This way we can use `self.meshset` everywhere in this class.